Library/Formula/wireshark.rb

```diff
@@ -6,6 +6,7 @@ class Wireshark < Formula
     url "https://www.wireshark.org/download/src/all-versions/wireshark-1.12.8.tar.bz2"
     mirror "https://1.eu.dl.wireshark.org/src/wireshark-1.12.8.tar.bz2"
     sha256 "357e0a4e49525d80cdc740bb16539fcdb526ad38cc2ed6cabedafc9bdee5c7e7"
+    revision 1
```

Comment:
Aye, you'll need to follow Baptiste's request here :). `revision` lines are always in the initial block, essentially.